repos/fdbt-site/src/pages/api/salesConfirmation.ts

```diff
@@ -104,14 +107,21 @@ export default async (req: NextApiRequestWithSession, res: NextApiResponse): Pro
             if (!myFaresEnabled || !exportEnabled) {
                 // if my fares or export isn't enabled we want to trigger the export lambda for a single
                 await triggerExport({ noc, paths: [filePath] });
+            } else {
+                if (dataFormat !== 'bods') {
```

Comment:
Think we probably want to follow the same logic as in insertDataToProductsBucketAndProductsTable:
(ticketType === 'geoZone' || dataFormat !== 'tnds')
Could probably do with having this as a function as it's used in a few places now

Reply:
Also minor: this could be an `else if (...)`